### DISCUSSION TITLE
Disable paragraph's margin bottom value

### DISCUSSION THREAD

**Discussion by BerlianoSurya** [twbs/bootstrap#38503] [Q&A]
Hi! I wanna ask something. Anyone know why is there a margin bottom value in `<p>`  tag from bootstrap? I want to disable it for my project and how to disable it? Please let me know. Ty.
![Screenshot 2023-04-25 162635](https://user-images.githubusercontent.com/90024019/234234500-26fe6cc9-d529-4f81-9b38-29c76c645879.png)
![Screenshot 2023-04-25 162838](https://user-images.githubusercontent.com/90024019/234234990-37013c25-5df8-4a58-a7c1-3c240643825b.png)

**Comment by nkdas91** [comment]
Hi @BerlianoSurya 

Override the `$paragraph-margin-bottom` variable in your custom scss file (where you import Bootstrap) to remove the `margin-bottom`.

```scss
$paragraph-margin-bottom: 0;
```

More info on how to override variables: https://getbootstrap.com/docs/5.3/customize/sass/

**Comment by thisisnotdevin** [comment]
I believe the previous comment should solve your problem, but you can also delete that line where you set `margin-bottom: $paragraph-margin-bottom`.

**Reply by zombiecatchersapk** [reply]
Bootstrap has a default margin bottom value of 1rem for the <p> tag to provide some spacing between paragraphs. However, if you want to disable it[,](https://zombiecatchersapk.com/zombie-catchers-weapons/) you can override it with your own CSS. Here's an example:

p {
  margin-bottom: 0; /* set margin bottom to 0 */
}
p {
  margin-bottom: 0; /* set margin bottom to 0 */
}
